core/frontend/src/components/wifi/DisconnectionDialog.vue

```diff
@@ -18,6 +18,25 @@
         </v-btn>
       </v-card-title>
 
+      <v-card-title
+        v-if="holds_ip_being_used"
+        class="justify-center"
+      >
+        <v-icon
+          color="white"
+          size="45"
+        >
+          mdi-alert-octagon
+        </v-icon>
+      </v-card-title>
+
+      <v-card-subtitle
+        v-if="holds_ip_being_used"
+        class="text-center mt-2 mb-2"
+      >
+        This IP address is currently being used to access BlueOS. Are you sure you want to disconnect?
```

Comment:
```suggestion
        This IP address is currently being used to access BlueOS. Deleting it could prevent you from accessing your vehicle. Are you sure you want to disconnect?
```